docs/manifest.json

```diff
@@ -39,6 +39,11 @@
           "title": "Upgrading",
           "description": "Learn how to upgrade Coder.",
           "path": "./install/upgrade.md"
+        },
+        {
+          "title": "Licensed Features",
```

Comment:
All the features are licensed, technically. I think it would be more clear to title this page `Enterprise`.

Reply:
And also "learn how to enable Enterprise features." ?

Reply:
Yeah I think that makes sense.